src/components/chip-set/chip-set.tsx

```diff
@@ -82,6 +82,12 @@ export class ChipSet {
     @Prop({ reflectToAttr: true })
     public searchLabel: string;
 
+    /**
+     * Label to display as `title` and `aria-label` for button that clears all chips
+     */
+    @Prop({ reflectToAttr: true })
+    public clearAllLabel: string;
```

Comment:
Is this to make it translatable or do we want consumers to be able to change the label to something completely different? 

If it's only for translation purposes, I would suggest to remove this for now and just have it in English. We need to discuss how to handle translations in Lime Elements at a broader scope before it becomes unmaintainable. I would prefer if we could solve it without having props for all translation strings on a component level. Thoughts? :slightly_smiling_face:

Reply:
> Is this to make it translatable or do we want consumers to be able to change the label to something completely different?

Ping @Kiarokh ☝️ 

> If it's only for translation purposes, I would suggest to remove this for now and just have it in English. We need to discuss how to handle translations in Lime Elements at a broader scope before it becomes unmaintainable. I would prefer if we could solve it without having props for all translation strings on a component level. Thoughts? 🙂

It seems we already have some temporary hardcoded translations in Lime Elements, used for the date-picker. If it's for translation, I would suggest adding the translations there for now.

I absolutely agree that we should discuss how to handle translations generally 👍

Reply:
> Ping @Kiarokh ☝️ 

thanks

> Is this to make it translatable

yes :) 

> handle translations in Lime Elements at a broader scope

would be awesome, decent, amazing, professional ;) 

> solve it without having props for all translation strings on a component level

you guys know the best how to do it optimally

Reply:
Great! I'll add some hardcoded translations 👍

Reply:
Ah, crap… no, that's just going to require us to send in the language code instead. I'm going with English only for now.

Unfortunately, since that property has already been merged and released, this is now becoming a breaking change 😭

Reply:
I decided to deprecate the property, and give it a default value in English. That way we can avoid making a breaking change for now.

Reply:
Sorry for being confused. The property was introduced in this PR, so I just removed it and used a hardcoded value in English instead.